src/components/interactive-map/interactive-map.tsx

```diff
@@ -263,8 +268,35 @@ const InteractiveMap: React.FC<InteractiveMapProps> = (props) => {
         .map(([provinceId, order]) => {
           const source = props.map.provinces.find((p) => p.id === provinceId);
           const target = props.map.provinces.find((p) => p.id === order.target);
+          const unit = props.units[provinceId];
+          const unitColor = unit ? props.nationColors[unit.nation] : "black"; // Default to black if no unit found
```

Comment:
Curious, can this happen in practice?

Reply:
I don't think really, unless it's a bug - but it helps identify issues when using StoryBook and is a nice fallback